shared/haml/user_header.haml

```diff
@@ -85,6 +85,9 @@
     .header_button.header_user#header_user_signin
       %span= I18n.t("#{loc_prefix}signin")
       .signin_callout_wrapper
+-# rubocop:disable CustomCops/DashboardDbUsage
```

Comment:
@davidsbailey -- is this something that we should make a ticket to remove? Also any guidance on how to avoid this usage of `current_user`? As Turner pointed out in the description, we use `current_user` in the header in multiple places

Reply:
Please don't introduce new violations of this type -- it should be possible to obtain the information you need from the user_type cookie.

Reply:
I'd be happy to help with this. Some info on it is here: https://docs.google.com/document/d/1-duNKVbhy14xrk95z7vYQOWXhQWjJ-YgcWE1sIbNTg8/edit?usp=drivesdk

Reply:
it should look something like this:
```
    cookie_key = environment_specific_cookie_name('_user_type')
    user_type = request.cookies[cookie_key]
    is_signed_out = user_type.nil?
```